features/css-selectors.yml

```diff
@@ -0,0 +1,32 @@
+name: CSS selectors (initial support)
+description: CSS selectors [TODO]
+spec: https://drafts.csswg.org/selectors-4/
+status:
+  compute_from: css.selectors.class
+compat_features:
+  - css.selectors.active
+  - css.selectors.active.non_a_elements
+  - css.selectors.child
+  - css.selectors.class
+  - css.selectors.descendant
+  - css.selectors.focus
+  - css.selectors.id
+  - css.selectors.list
+  - css.selectors.next-sibling
+  - css.selectors.root
+  - css.selectors.type
+  - css.selectors.universal
+  - css.selectors.type.namespaces
+  - css.selectors.universal.namespaces
+  - css.selectors.target
+  - css.selectors.hover
+  - css.selectors.attribute
+  - css.selectors.subsequent-sibling
+  - css.selectors.checked
+  - css.selectors.disabled
+  - css.selectors.empty
+  - css.selectors.empty.matches_whitespace
+  - css.selectors.enabled
+  - css.selectors.lang
+  - css.selectors.not
+  - css.selectors.not.selector_list
```

Comment:
`:not` might be its own thing—it "feels" newer to me 🤷

Reply:
It also feels new to me for some reason, strange because it's been supported forever. The selector_list subfeature, however, is much newer. So I agree with Daniel about moving them both to a separate feature because of this.